index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+  <link href="https://fonts.googleapis.com/css2?family=Inter:wght@300&display=swap" rel="stylesheet">
+</head>
+<body>
+  
+    <header class="header">
+      <h3 class="head">Welcome</h3>
+      <nav class="menu-bar">
+        <img src="images/Normal Button/Enabled.png" alt="">
+      </nav>
+    </header>
+    <section class="head-section">
+    <div class="hey-there">
+<h2 class="hey-text">Hey there. I’m Mariane
+ <p class="hey-text2"> I’m a software developer</p></h2>
```

Comment:
- [ ] The main title of the page is wrapped with the `h1` tag for better SEO performance. Kindly change the tag of the main title. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info.
 - [ ] If there is a different text inside a paragraph or heading text, it is usually used the `span` tag for better HTML semantics. Kindly change the `p` tag to a `span`.